package.json

```diff
@@ -131,15 +126,14 @@
     "minimist": "^1.2.6",
     "mocha": "^9.0.1",
```

Comment:
Note that Mocha and Chai are still needed for our slow test suite. I'll work on removing those in a follow-up PR.